components/sup/src/config.rs

```diff
@@ -120,5 +120,33 @@ impl fmt::Display for GossipListenAddr {
 #[derive(Default, Debug, PartialEq, Eq)]
 pub struct Config {
     // Currently only used by `ServiceConfig`
-    pub service_config_http_listen: http_gateway::ListenAddr,
+    pub service_config_http_listen: HttpGatewayListenAddr,
+    // Currently only used by `Package`
+    pub package_config_from: Option<String>,
```

Comment:
This is removed in the master branch. You'll want to make sure it's dropped here, too